example/.gitattributes

```diff
@@ -0,0 +1,7 @@
+# Do not try and format generated code
+pnpm-lock.yaml linguist-generated
+
+# Any of the below attributes can be set to ignore a file
+src/subdir/rules_lint_ignored.js rules_lint_ignored
```

Comment:
nit: seems like the convention is "kebab case" so I don't think we should be different and use snake case, will be harder to remember what to type

Reply:
Done